src/index.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+const state = {
+  temperature: 0,
+  landscape: document.getElementById('landscape'),
+  location: '',
+  skyType: document.getElementById('sky-type'),
+  skyDisplay: document.getElementById('sky-landscape'),
+};
+
+const changeTemperatureEnvironment = (displayTemperatureEl) => {
+  displayTemperatureEl.className = 'padding-5';
+
+  const landscape = {
+    cold: '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲',
+    warm: '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃',
+    veryWarm: '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷',
+    hot: '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂',
+  };
+
+  if (state.temperature <= 49) {
+    displayTemperatureEl.classList.add('teal');
+  } else if (state.temperature <= 59) {
+    displayTemperatureEl.classList.remove('teal');
+    displayTemperatureEl.classList.add('green');
+    state.landscape.textContent = landscape.cold;
+  } else if (state.temperature <= 69) {
+    displayTemperatureEl.classList.remove('green');
+    displayTemperatureEl.classList.add('yellow');
+    state.landscape.textContent = landscape.warm;
+  } else if (state.temperature <= 79) {
+    displayTemperatureEl.classList.remove('yellow');
+    displayTemperatureEl.classList.add('orange');
+    state.landscape.textContent = landscape.veryWarm;
+  } else {
+    displayTemperatureEl.classList.remove('orange');
+    displayTemperatureEl.classList.add('red');
+    state.landscape.textContent = landscape.hot;
+  }
```

Comment:
Right now we manage both adding and removing class names. Another option would be to overwrite the `className` attribute with a string that contained both the padding class name and the color class name separated by a space. That way we always have the `'padding-5'` class name on the element and we only worry about changing the color class name we want it to be now, we don't need to manually remove the previous class.
```javascript
  const paddingClass = 'padding-5';

  if (state.temperature <= 49) {
    displayTemperatureEl.className = `${paddingClass} teal`;
  } else if (state.temperature <= 59) {
    displayTemperatureEl.className = `${paddingClass} green`;
    state.landscape.textContent = landscape.cold;
  } else if (state.temperature <= 69) {
    displayTemperatureEl.className = `${paddingClass} yellow`;
    state.landscape.textContent = landscape.warm;
  } ... // and so on for the other colors
```